git-multimail/git_multimail.py

```diff
@@ -2677,11 +2677,18 @@ def get_emailprefix(self):
         if emailprefix is not None:
             emailprefix = emailprefix.strip()
             if emailprefix:
-                return emailprefix + ' '
-            else:
-                return ''
+                emailprefix += ' '
         else:
-            return '[%s] ' % (self.get_repo_shortname(),)
+            emailprefix = '[%(repo_shortname)s] '
+        try:
+            return emailprefix % {'repo_shortname': self.get_repo_shortname()}
+        except:
+            sys.stderr.write(
+                '*** Invalid multimailhook.emailPrefix: %s\n' % emailprefix +
+                '*** %s' % (sys.exc_info()[1]) +
+                "*** Only the '%(repo_shortname)s' placeholder is allowed\n"
+                )
+            sys.exit(1)
```

Comment:
We normally avoid `sys.exit`, to let the caller catch potential exception. This looks like a `ConfigurationException`.